notary-iroha-integration-test/src/main/kotlin/integration/helper/IrohaAccountHelper.kt

```diff
@@ -35,6 +35,7 @@ class IrohaAccountHelper(private val irohaAPI: IrohaAPI, private val peers: Int
     /** Notary account */
     val notaryAccount by lazy { createNotaryAccount() }
 
+    val clientStorageAccount by lazy { createTesterAccount("client_accounts").accountId }
```

Comment:
Don't forget to set a role for this account.

Reply:
It's created in the domain default role of which is convenient for the account